blog/models.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    class Meta:
+        default_related_name = "users"
+        verbose_name = "user"
+        verbose_name_plural = "users"

```

Comment:
Are you sure about this, i think it would be the same names as default?

Reply:
Thank you! Totaly agree.